src/data/polynomial/derivative.lean

```diff
@@ -214,5 +214,31 @@ le_antisymm
 
 end domain
 
+section char_zero
+
+variables [field R] [char_zero R]
```

Comment:
I don't think you need `field R` for this lemma.

@jcommelin suggested generalising this to `integral_domain R`, so (again) I think this lemma would be fine in the `domain` section.

Reply:
done